statement.go

```diff
@@ -7,6 +7,11 @@ import (
 	"database/sql/driver"
 )
 
+// SnowflakeStmt provides an API for methods exposed to the clients
+type SnowflakeStmt interface {
```

Comment:
almost same name as snowflakeStmt, will it create confusion?

Reply:
This is the same as [SnowflakeRows/snowflakeRows](https://github.com/snowflakedb/gosnowflake/blob/master/rows.go#L32) and `[SnowflakeResult/snowflakeResult](https://github.com/snowflakedb/gosnowflake/blob/master/result.go#L17)`. I followed the same format to keep it consistent.